### PR TITLE
SubviewCollection should provide a way to append each subview element to the view

### PR DESCRIPTION
**Pull request by justinvdm**
This ticket aims to solve what #402 was meant to solve.

### CODE REVIEW

File: go/base/static/js/test/tests/components/plumbing/diagrams.test.js [hunk 1]
```diff
@@ -19,123 +19,6 @@ describe("go.components.plumbing.diagrams", function() {
...
-          assert.equal(a1L2_b2R2, connection);
-
-          done();
-        });
-
-        jsPlumb.detach(a1L2_b2R2.plumbConnection);
-      });
-    });
-  });
-
```
Comment (hodgestar) [inline go/base/static/js/test/tests/components/plumbing/diagrams.test.js:138]: How come all these tests went away?

Comment (justinvdm) [inline go/base/static/js/test/tests/components/plumbing/diagrams.test.js:138]: Ya, this is unrelated. After noticing this last night, I pasted a gist link on irc, but I think it was buried beneath git commit messages:

> 20:47 < justinvdm> margle/Hodgestar: https://gist.github.com/justinvdm/5852425
> 20:48 < justinvdm> I forgot to remove this stuff when moved DiagramViewConnections into go.components.plumbing.connections and called it DiagramConnectionGroup

Comment (hodgestar) [inline go/base/static/js/test/tests/components/plumbing/diagrams.test.js:138]: Ah! Np.

**Comment by hodgestar** [comment]
:+1:
